rules/symfony5/src/Rector/New_/PropertyAccessorCreationBooleanToFlagsRector.php

```diff
@@ -62,43 +63,41 @@ public function refactor(Node $node): ?Node
         }
 
         $isTrue = $this->isTrue($node->args[0]->value);
-        $flags = $this->prepareFlags($isTrue);
-        $node->args[0] = $this->createArg($flags);
+        $bitwiseOr = $this->prepareFlags($isTrue);
+        $node->args[0] = $this->createArg($bitwiseOr);
 
         return $node;
     }
 
+    private function shouldSkip(New_ $new): bool
+    {
+        if (! $new->class instanceof Name) {
+            return true;
+        }
+
+        if (! $this->isName($new->class, 'Symfony\Component\PropertyAccess\PropertyAccessor')) {
+            return true;
+        }
+        return ! $this->isBool($new->args[0]->value);
+    }
+
     private function prepareFlags(bool $currentValue): BitwiseOr
     {
-        $magicGet = $this->createClassConstFetch('Symfony\Component\PropertyAccess\PropertyAccessor', 'MAGIC_GET');
+        $classConstFetch = $this->createClassConstFetch(
```

Comment:
Name `classConstFetch` can be misleading as we have multiple of them created here (one line below we have `magicSet` for example).